src/style.css

```
@@ -1 +1,97 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  /* list-style: none;
+  flex-direction: row; */
+  justify-content: space-around;
```

```suggestion
  justify-content: space-between;
```
Для відступів по бокам задай паддінги по 50 пікселів для хедера